roles/nginx/templates/cloudwatch-vhost.json.j2

```diff
@@ -6,11 +6,20 @@
           {
             "file_path": "{{ domain.access_log }}",
             "log_group_name": "nginx-access",
+{% if nginx.log_group_prefix is defined and nginx.log_group_prefix|length %}
+            "log_group_name": "{{nginx.log_group_prefix}}nginx-access",
```

Comment:
This should probably have spaces inside the braces. (Sorry!)

Reply:
yep